code/write_excel/95_Source_Excel_4.R

```diff
@@ -24,13 +23,13 @@ year_xx <- year(cut_off_date)
 year_ww <- year_xx - 1
 year_vv <- year_xx - 2
 year_yy <- year_xx + 1
-year_3 <- year_xx - 3
-year_5 <- year_xx - 5
+year_uu <- year_xx - 3
```

Comment:
I did think about sticking to my alphabet system, but then was too lazy to figure out x-5...